src/components/Creatives/CreativeList.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  CreativeFragment,
+  useAdvertiserCreativesQuery,
+} from "graphql/creative.generated";
+import { uiTextForCreativeTypeCode } from "user/library";
+import { CardContainer } from "components/Card/CardContainer";
+import { useAdvertiser } from "auth/hooks/queries/useAdvertiser";
+import { ErrorDetail } from "components/Error/ErrorDetail";
+import MiniSideBar from "components/Drawer/MiniSideBar";
+import { Box, Link, List, Typography } from "@mui/material";
+import { Status } from "components/Campaigns/Status";
+import { Link as RouterLink } from "react-router-dom";
+import {
+  DataGrid,
+  GridColDef,
+  GridToolbarColumnsButton,
+  GridToolbarContainer,
+  GridToolbarFilterButton,
+  GridToolbarQuickFilter,
+} from "@mui/x-data-grid";
+import { CreativeStatusSwitch } from "components/Creatives/CreativeStatusSwitch";
+
+const ALLOWED_TYPES = ["notification_all_v1", "inline_content_all_v1"];
+export function CreativeList() {
+  const { advertiser } = useAdvertiser();
+  const { data, error, loading } = useAdvertiserCreativesQuery({
+    variables: {
+      advertiserId: advertiser.id,
+    },
+    pollInterval: 60_000,
+  });
+
+  const columns: GridColDef<CreativeFragment>[] = [
+    {
+      field: "switch",
+      type: "actions",
+      headerName: "On / Off",
+      renderCell: ({ row }) => <CreativeStatusSwitch creative={row} />,
+      filterable: false,
+      sortable: false,
+    },
+    {
+      field: "name",
+      type: "string",
+      headerName: "Name",
+      renderCell: ({ row }) => (
+        <Link
+          underline="none"
+          variant="body1"
+          component={RouterLink}
+          to={`/user/main/creative/${row.id}`}
+          replace
+        >
+          {row.name}
+        </Link>
+      ),
+      flex: 1,
+      minWidth: 100,
+      maxWidth: 400,
+    },
+    {
+      field: "type",
+      headerName: "Type",
+      valueGetter: ({ row }) => uiTextForCreativeTypeCode(row.type),
+      align: "left",
+      width: 200,
+    },
+    {
+      field: "content",
+      headerName: "Content",
+      renderCell: ({ row }) => <CreativePayloadList creative={row} />,
+      flex: 1,
+      sortable: false,
+    },
```

Comment:
The filter on this doesn't work for me - if I filter based on a title of a creative, it doesn't match. I think you just need to return plan text in `valueGetter` for that to work?